05-script.md

```diff
@@ -208,9 +207,11 @@ to handle the case of parameters containing spaces
 Here's an example:
 
 ~~~ {.bash}
-$ cat sorted.sh
+$ nano sorted.sh
```

Comment:
The idea with "cat" is for it to correspond to the {.output} below. we're not editing the file, just viewing it.